EST.MIT.Payment.Function/Validation/ValidateRequest.cs

```diff
@@ -168,14 +178,14 @@ public static class ValidateRequest
             }
         },
         ""required"": [
-            ""invoices""
+            ""paymentRequestsBatches""
         ]
         }";
 
     public static bool IsValid(string importRequest)
     {
         var schema = JSchema.Parse(schemaJson);
-        var parseImportRequest = JObject.Parse(importRequest);
+        var parseImportRequest = JObject.Parse(importRequest);  
```

Comment:
Being picky, I think there's a trailing space here now that is causing a 'diff' on this line to show. Be careful not to introduce white space!

Reply:
resolved